src/cljs/main/broadfcui/page/workspace/monitor/submission_details.cljs

```diff
@@ -27,78 +28,93 @@
-       :filter-groups
-       (vec (cons {:text "All" :pred (constantly true)}
-                  (map (fn [status] {:text status :pred #(= status (:status %))})
-                       moncommon/wf-all-statuses)))
-       :data (:workflows props)
-       :->row (fn [row]
-                [(:workflowId row)
-                 (:workflowEntity row)
-                 (:statusLastChangedDate row)
-                 (:status row)
-                 (:messages row)
-                 row])}])
+      {:ref "table"
+       :data (:filtered-data @state)
+       :body
+       {:empty-message "No Workflows"
+        :style table-style/table-heavy
+        :behavior {:fixed-column-count 1}
+        :columns [
+                   {:id "view"
+                    :column-data :workflowId
+                    :initial-width 50
+                    :as-text (constantly "View workflow details")
+                    :hidden? true
+                    :resizable? false :sortable? false :filterable? false
+                    :render
+                    (fn [id]
+                      (when id
+                        (style/create-link
+                         {:text "View"
+                          :href (nav/get-link :workspace-workflow
+                                              (:workspace-id props)
+                                              (:submission-id props)
+                                              id)})))}
+                   {:header "Data Entity" :initial-width 200
+                    :column-data :workflowEntity
+                    :as-text
+                    (fn [entity]
+                      (str (:entityName entity) " (" (:entityType entity) ")"))
+                    :sort-by :text}
+                   {:header "Last Changed" :initial-width 280
+                    :column-data :statusLastChangedDate
+                    :sort-initial :desc
+                    :as-text moncommon/render-date}
+                   {:header "Status" :initial-width 120
+                    :column-data :status
+                    :render (fn [status]
+                              [:div {}
+                               (moncommon/icon-for-wf-status status)
+                               status])}
+                   {:header "Messages" :initial-width 300
+                    :column-data :messages
+                    :render (fn [message-list]
+                              [:div {}
+                               (map (fn [message]
+                                      [:div {} message])
+                                    message-list)])}
+                   {:header "Workflow ID" :initial-width 300
+                    :as-text :workflowId :sort-by :text
+                    :render
+                    (fn [workflow]
+                      (let [{:keys [submission-id bucketName]} props
+                            inputs (second (second (first (:inputResolutions workflow))))
```

Comment:
Could potentially use a threading macro here like 

`(-> workflow :inputResolutions first second second)`

But I guess the way it is now is pretty clear, so it's not necessary